.github/workflows/ci.yml

```diff
@@ -1,38 +1,293 @@
+          docker-password: ${{ secrets.DOCKERHUB_TOKEN }}
+          docker-images: >-
+            springcloudstream/composed-http-transformer-kotlin-kafka:0.0.1-SNAPSHOT
+
+  stream-dev-guides-composed-http-transformer-rabbitmq:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ./.github/actions/build-sample-app
+        with:
+          app-dir: 'dataflow-website/stream-developer-guides/feature-guides/streams/function-composition/composed-http-transformer-rabbitmq'
+          mvn-build-commands: '-B clean install jib:dockerBuild'
+          artifactory-publish: ${{ inputs.maven-build-only != 'true' }}
+          jf-artifactory-spring: ${{ secrets.JF_ARTIFACTORY_SPRING }}
+          docker-push: ${{ inputs.maven-build-only != 'true' }}
+          docker-username: ${{ secrets.DOCKERHUB_USERNAME }}
+          docker-password: ${{ secrets.DOCKERHUB_TOKEN }}
+          docker-images: >-
+            springcloudstream/composed-http-transformer-rabbitmq:0.0.1-SNAPSHOT            
+
+  stream-dev-guides-composed-http-transformer-kotlin-rabbitmq:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ./.github/actions/build-sample-app
+        with:
+          app-dir: 'dataflow-website/stream-developer-guides/feature-guides/streams/function-composition/composed-http-transformer-kotlin-rabbitmq'
+          mvn-build-commands: '-B clean install jib:dockerBuild'
+          artifactory-publish: ${{ inputs.maven-build-only != 'true' }}
+          jf-artifactory-spring: ${{ secrets.JF_ARTIFACTORY_SPRING }}
+          docker-push: ${{ inputs.maven-build-only != 'true' }}
+          docker-username: ${{ secrets.DOCKERHUB_USERNAME }}
+          docker-password: ${{ secrets.DOCKERHUB_TOKEN }}
+          docker-images: >-
+            springcloudstream/composed-http-transformer-kotlin-rabbitmq:0.0.1-SNAPSHOT
+
+  task-demo-metrics-prometheus:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ./.github/actions/build-sample-app
+        with:
+          app-dir: 'monitoring-samples/task-apps/task-demo-metrics-prometheus'
+          mvn-build-commands: '-B clean install docker:build'
+          artifactory-publish: ${{ inputs.maven-build-only != 'true' }}
+          jf-artifactory-spring: ${{ secrets.JF_ARTIFACTORY_SPRING }}
+          docker-push: ${{ inputs.maven-build-only != 'true' }}
+          docker-username: ${{ secrets.DOCKERHUB_USERNAME }}
+          docker-password: ${{ secrets.DOCKERHUB_TOKEN }}
+          docker-images: >-
+            springcloudtask/task-demo-metrics-prometheus:0.0.4-SNAPSHOT
+
+  scenario-task:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ./.github/actions/build-sample-app
+        with:
+          app-dir: 'scenario-task'
+          mvn-build-commands: '-B clean install spring-boot:build-image -Dspring-boot.build-image.imageName=springcloudtask/scenario-task:0.0.1-SNAPSHOT'
```

Comment:
Did we want to make the sample app versions to be 2.0.0-SNAPSHOT?

Reply:
Ditto https://github.com/spring-cloud/spring-cloud-dataflow-samples/pull/159#discussion_r789850455